RevenueCatUI/Resources/icons.xcassets/paywall-first-step-macOS.imageset/Contents.json

```diff
@@ -0,0 +1,22 @@
+{
+  "images" : [
+    {
+      "filename" : "image-export-curve-macos.png",
+      "idiom" : "universal",
+      "scale" : "1x"
+    },
+    {
+      "filename" : "image-export-curve-macos@2x.png",
+      "idiom" : "universal",
+      "scale" : "2x"
+    },
+    {
+      "idiom" : "universal",
+      "scale" : "3x"
+    }
+  ],
+  "info" : {
+    "author" : "xcode",
+    "version" : 1
+  }
+}
```

Comment:
this probably doesn't belong here, can't we use the same solution as with localize+bundle?

Reply:
ah, this is preview-only, cool

Reply:
Yes, I only put two for the preview.